test/e2e/storage/csi_volumes.go

```diff
@@ -139,7 +141,12 @@ var _ = utils.SIGDescribe("CSI Volumes", func() {
 				driver.cleanupCSIDriver()
 			})
 
-			It("should provision storage", func() {
+			It("should provision storage and allow driver update", func() {
```

Comment:
i understand the want that one test is not strict subset of another; however, by doing this we have lost granularity of signals. When this test fails we don't have a simple signal whether the driver is failing to work in general or if it is failing to update.

Though I could be convinced this is ok because I guess you can look at error logs to see which error it really is. 
@msau42 @jsafrane what do you think?

Reply:
would prefer two different test cases.  One for basic functionality, one for testing update

Reply:
Ack, looking at the test structure now, I need to create a separate test anyway.